about.css

```diff
@@ -0,0 +1,266 @@
+.background-aboutPage {
+  background: url(./assets/img/about-banner.png);
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-color: #f7f7f7;
+  width: 100%;
+  max-width: 1300px;
+  margin: auto;
+  height: 620px;
+  display: flex;
+}
+
+.intro-header-container-aboutPage {
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.image-title-aboutPage {
+  width: 60%;
+  margin-top: 90px;
+}
+
+.intro-text-aboutPage {
+  border: 1px solid lightgray;
+  width: 75%;
+  padding: 29px;
+  margin-top: 25px;
+  background-color: #fff;
+  margin-bottom: 35px;
+}
+
+.intro-text-aboutPage span {
+  color: gray;
+  font-size: 13px;
+  font-weight: 900;
+}
+
+.text-aboutPage {
+  font-weight: 900;
+  font-size: 14px;
+  color: #545454;
+  margin: 0 6px;
+}
+
+.emailto {
+  color: black;
+  font-weight: 900;
+}
+
+.background-MainProgram-aboutPage {
+  background-color: #fff;
+  width: 100%;
+  max-width: 1300px;
+  margin: auto;
+}
+
+.mainProgram-container-title-h1-aboutPage {
+  color: black;
+  font-size: 20px;
+  font-weight: 900;
+}
+
+.mainProgram-p-aboutpage {
+  text-align: center;
+  font-size: 12px;
+  font-weight: 600;
+}
+
+.winning-logo {
+  margin: 35px 0;
+}
+
+.winning-logo img {
+  border: 0.1px solid #ececec;
+}
+
+.border-top {
+  border-top: 1px solid lightgray;
+}
+
+.mainProgram-container-title-h1-summit {
+  color: black;
+  font-size: 20px;
+  font-weight: 900;
+  text-shadow: 0.1px 0 black;
+}
+
+.hr_submit_logo {
+  width: 7% !important;
+}
+
+.pastSubmits_container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px 0;
+}
+
+.background-past1 {
+  background-image: url(./assets/img/past1.jpg);
+  width: 275px;
+  height: 135px;
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 0 10px;
+}
+
+.background-past2 {
+  background-image: url(./assets/img/past2.jpg);
+  width: 275px;
+  height: 135px;
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 0 10px;
+}
+
+.color-background-past {
+  background-color: rgba(141, 49, 39, 0.7);
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.pastSubmits-Year {
+  font-size: 23px;
+  font-weight: 900;
+  color: #e5e5e5;
+}
+
+.pastSubmits-location {
+  font-size: 12px;
+  color: #e5e5e5;
+  text-shadow: 0.1px 0 #e5e5e5;
+}
+
+@media only screen and (max-width: 768px) {
```

Comment:
- [ ] Please as requested by the previous code reviewer, make sure you implement the mobile first approach, for the image you are complaining that it is affecting the responsiveness, define the image width for the mobile version, then use media queries to define the image width for the desktop version not vice versa.

Reply:
Thanks for the feedback!

I have now understood the mobile-first approach, now I am only using:

`@media only screen and (min-width: 768px)`

to convert from the native mobile to desktop.

Now you can't find any `@media only screen and (max-width: 768px)` to adjust desktop to mobile..

This is now a fully mobile-first project!! Thanks!!